### PR TITLE
Fix compilation for GTClang with LLVM 9

### PR DESCRIPTION
**Pull request by lukasm91**
## Technical Description

Fix compilation with clang 9. I can still not use gtclang built with llvm 9, but at least it compiles.

### Resolves / Enhances

Regression from #344

### CODE REVIEW

**Comment by lukasm91** [comment]
launch jenkins

**Comment by lukasm91** [comment]
@havogt Are you sure?
https://github.com/llvm-mirror/clang/blob/release_90/include/clang/Frontend/CompilerInvocation.h#L154

**Comment by havogt** [comment]
No ;)
Probably I added this while being on master...

File: gtclang/src/gtclang/Support/ClangCompat/CompilerInvocation.h [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #include "llvm/Option/Option.h"
 
 namespace gtclang::clang_compat::CompilerInvocation {
-#if CLANG_VERSION_MAJOR < 9
+#if CLANG_VERSION_MAJOR <= 9
```
Comment (havogt) [inline gtclang/src/gtclang/Support/ClangCompat/CompilerInvocation.h]: Can you change to < 10? I think that's the pattern I usually used.

Comment (lukasm91) [inline gtclang/src/gtclang/Support/ClangCompat/CompilerInvocation.h]:
```suggestion
#if CLANG_VERSION_MAJOR < 10
```

File: gtclang/src/gtclang/Support/ClangCompat/FileUtil.h [hunk 2]
```diff
@@ -21,7 +21,7 @@
 #include "clang/Basic/Version.h"
 
 namespace gtclang::clang_compat::FileUtil {
-#if CLANG_VERSION_MAJOR < 9
+#if CLANG_VERSION_MAJOR <= 9
```
Comment (lukasm91) [inline gtclang/src/gtclang/Support/ClangCompat/FileUtil.h]:
```suggestion
#if CLANG_VERSION_MAJOR < 10
```

File: gtclang/src/gtclang/Support/ClangCompat/CompilerInvocation.h (hunk 1 above)
Comment (lukasm91) [reply gtclang/src/gtclang/Support/ClangCompat/CompilerInvocation.h]: you're right

**Comment by lukasm91** [comment]
launch jenkins

**Pull request merged**